### PR TITLE
[Feature][Resource Center] Add support for Huawei Cloud OBS

### PR DESCRIPTION
**Pull request by ediconss**
## Purpose of the pull request

Add support for Huawei Cloud OBS 

## Verify this pull request
UT and manually tested

1. configure in the common.properties

```
 resource.huawei.cloud.access.key.id=<your-access-key-id>
 resource.huawei.cloud.access.key.secret=<your-access-key-secret>
 resource.huawei.cloud.obs.bucket.name=dolphinscheduler
 resource.huawei.cloud.obs.endpoint=obs.cn-southwest-2.huaweicloud.com

```
2. Upload files and directory in the resource center
obs

### CODE REVIEW

**Review by SbloodyS — COMMENTED** [review]
Please add it to the docs too. @ediconss

**Comment by ediconss** [comment]
> Please add it to the docs too. @ediconss

this doc file?
docs/docs/zh/guide/resource/configuration.md

**Comment by SbloodyS** [comment]
> > Please add it to the docs too. @ediconss
> 
> this doc file? docs/docs/zh/guide/resource/configuration.md

Both `docs/docs/zh/guide/resource/configuration.md` and `docs/docs/en/guide/resource/configuration.md`.

File: docs/docs/en/guide/resource/configuration.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ data.basedir.path=/tmp/dolphinscheduler
 # resource view suffixs
 #resource.view.suffixs=txt,log,sh,bat,conf,cfg,py,java,sql,xml,hql,properties,json,yml,yaml,ini,js
 
-# resource storage type: HDFS, S3, OSS, GCS, ABS, NONE
+# resource storage type: HDFS, S3, OSS, GCS, ABS, OBS, NONE
```
Comment (Radeity) [inline docs/docs/en/guide/resource/configuration.md]: Not related to this PR, but can you help make this comment consistent with chinese-doc :D

File: dolphinscheduler-bom/pom.xml [hunk 2]
```diff
@@ -895,6 +896,12 @@
                 <artifactId>protobuf-java</artifactId>
                 <version>${protobuf.version}</version>
             </dependency>
+
+            <dependency>
+                <groupId>com.huaweicloud</groupId>
+                <artifactId>esdk-obs-java-bundle</artifactId>
+                <version>${esdk-obs.version}</version>
+            </dependency>
```
Comment (rickchengx) [inline dolphinscheduler-bom/pom.xml]: please add the license of your 3-party dependency
FYI: https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/DS-License.md

File: docs/docs/en/guide/resource/configuration.md (hunk 1 above)
Comment (ediconss) [reply docs/docs/en/guide/resource/configuration.md]: sorry i can't understand what you mean，i think OBS is new resource storage type, it is related

Comment (Radeity) [reply docs/docs/en/guide/resource/configuration.md]:
```suggestion
# resource storage type: LOCAL, HDFS, S3, OSS, GCS, ABS, OBS, NONE
```
Also, please add type NONE in chinese doc :D

Comment (ediconss) [reply docs/docs/en/guide/resource/configuration.md]: all to  # resource storage type: LOCAL, HDFS, S3, OSS, GCS, ABS, OBS, NONE 
it's ok?

Comment (Radeity) [reply docs/docs/en/guide/resource/configuration.md]: Yes.

**Comment by rickchengx** [comment]
Hi, @ediconss could you please give some screenshots of the files uploaded through the resource center in OBS
* FYI: something like #13562

**Comment by ediconss** [comment]
> Hi, @ediconss could you please give some screenshots of the files uploaded through the resource center in OBS
> 
> * FYI: something like [[Feature-13428][Resource Center] Add support for Google Cloud Storage in the resource center #13562](https://github.com/apache/dolphinscheduler/pull/13562)

sure

**Review by rickchengx — APPROVED** [review]
Good work @ediconss 
LGTM if CI pass

**Pull request merged**